src/mdx/_Code.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Highlight, { defaultProps as highlightProps } from 'prism-react-renderer';
+import classNames from 'classnames/bind';
+import styles from './MarkdownTags.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Children for the component.
+   */
+  children: PropTypes.node,
+  /**
+   * ClassName for the component.
+   */
+  className: PropTypes.string,
+
+};
+
+const Code = ({ children, className }) => {
+  const language = (className || '').replace(/language-/, '');
```

Comment:
What is this doing?

Reply:
classname is passed in as `language-js` or `language-ts`, this removes the `language-` portion. classname can also be undefined, and if thats the case we can't call replace on an undefined string, so i sub in an empty string.